gatsby-config.js

```diff
@@ -5,8 +5,18 @@ module.exports = {
       'This repo contains an example business website that is built with Gatsby, and Netlify CMS.It follows the JAMstack architecture by using Git as a single source of truth, and Netlify for continuous deployment, and CDN distribution.',
   },
   plugins: [
+    'gatsby-plugin-sharp',
+    'gatsby-transformer-sharp',
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-sass',
+    {
+      // keep as first gatsby-source-filesystem plugin for gatsby image support
```

Comment:
Invalid comment due to copying, and the comment clearly states that for image support, images needs to be first source filesystem plugin. I haven't look too deep into this, but maybe just to be safe this added block can be second one, after the images?